libs/common/async.go

```diff
@@ -151,11 +151,11 @@ func Parallel(tasks ...Task) (trs *TaskResultSet, ok bool) {
 			var val, err, abort = task(i)
 			// Send val/err to taskResultCh.
 			// NOTE: Below this line, nothing must panic/
-			taskResultCh <- TaskResult{val, err}
+			taskResultCh <- TaskResult{val, abort}
```

Comment:
Doesn't this change the logic?

Reply:
I meant to change line 151 instead. The types haven't changed, just the variable names.